resources/views/app/includes/contents/channels.blade.php

```diff
@@ -1,7 +1,20 @@
 <div class="row">
 
 <!-- Side Nav -->
-    @include('app.includes.sections.sidenav')
+    <div class="col s3">
+        <div class="hide-on-small-only">
+            <div class="collection">
+                <a href="/channels" class="collection-item">Channels <span class="new badge">4</span></a>
+                <a href="/episodes" class="collection-item" id="view-all-episodes">See all episodes
+                    <span class="badge">10+</span></a>
+                    @can('guest', Auth::check())
+                    <a href="{{ URL::to('about') }}" class="collection-item">About</a>
+                    <a href="{{ URL::to('privacypolicy') }}" class="collection-item">Privacy Policy</a>
+                    <a href="{{ URL::to('faqs') }}" class="collection-item">FAQs</a>
+                @endcan
```

Comment:
**Code Quality - Nitpick**

As a sidenav, it should be moved into a partial file. Then, the partial file should be included here.